src/App.tsx

```diff
@@ -1,51 +1,37 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { NavbarLink } from './components/NavbarLink';
+import { HomePage } from './pages/HomePage';
+import { TabsPage } from './pages/TabsPage';
+import { NotFoundPage } from './pages/NotFoundPage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
-
-export const App = () => (
+export const App: React.FC = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
       className="navbar is-light is-fixed-top is-mobile has-shadow"
       data-cy="Nav"
     >
       <div className="container">
         <div className="navbar-brand">
```

Comment:
you can move NavBar to the separate component

Reply:
Agree, will be better to move it to separate component